k8s/aat/common/ccd/definition-store-api.yaml

```
@@ -29,6 +39,8 @@ spec:
         ELASTIC_SEARCH_FAIL_ON_IMPORT: true
         DEFINITION_STORE_DB_MAX_POOL_SIZE: 25
         DEFINITION_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_admin,jui_webapp,pui_webapp,aac_manage_case_assignment,em_hrs_api
+        DEFINITION_STORE_DB_NAME: ccd_definition_store
+        DEFINITION_STORE_DB_HOST: ccd-definition-store-api-aat-db-v11-perftest.postgres.database.azure.com
```

```suggestion
        DEFINITION_STORE_DB_HOST: ccd-definition-store-api-db-v11-aat.postgres.database.azure.com
```
Please check this value